ch04.md

```diff
@@ -1,6 +1,6 @@
 # Chapter 04: Currying
 
-## Can't live if livin' is without you
+## Can't Live If Livin' Is without You
```

Comment:
without -> Without ?

Reply:
As described in https://github.com/MostlyAdequate/mostly-adequate-guide/issues/380#issuecomment-364848439 I propose that we keep “without” lowercased.